File: src/main/java/SpartJavaSample/LoginController.java
package SpartJavaSample;

import spark.Request;
import spark.Response;

import java.util.Map;

public class LoginController implements Controllable
{
    @Override
    public boolean get(Request request, Response response, Map<String, Object> model)
    {
        model.put("test", 44);

        return true;
    }

    @Override
    public boolean post(Request request, Response response, Map<String, Object> model)
    {
        request.session(); // Create session

        response.redirect("/authorize");

        return false;
    }
}

File: src/main/kotlin/SpartJavaSample/LoginController.kt
package SpartJavaSample

import spark.Request
import spark.Response

public class LoginController : Controllable()
{
    public override fun get(request: Request, response: Response): ControllerResult =
            ControllerResult(model = mapOf("test" to 44))

    public override fun post(request: Request, response: Response): ControllerResult
    {
        request.session() // Create session

        response.redirect("/authorize")

        return ControllerResult(continueProcessing = false)
    }
}
